src/v3/src/transformer/password/transformEnrollPasswordAuthenticator.ts

```diff
@@ -154,22 +154,6 @@ export const transformEnrollPasswordAuthenticator: IdxStepTransformer = ({
   dataSchema.fieldsToValidate.push('confirmPassword');
   dataSchema.fieldsToValidate.push('passwordMatchesValidation');
 
-  // Controls live field change validation
-  dataSchema.confirmPassword = {
-    validate: (data: FormBag['data']) => {
-      const confirmPw = data.confirmPassword;
-      if (!confirmPw) {
-        return [{
-          name: 'confirmPassword',
-          class: 'ERROR',
-          message: loc('model.validation.field.blank', 'login'),
-          i18n: { key: 'model.validation.field.blank' },
-        }];
-      }
-      return undefined;
-    },
-  };
```

Comment:
This was hiding in plain sight and didnt notice it on my last fix. 😢

Reply:
re: the comment, we no longer have live field validation, it only validates on blur now, so this is why this is no longer needed.